sbin/build.sh

```diff
@@ -389,7 +389,12 @@ configureVersionStringParameter() {
       addConfigureArg "--with-version-opt=" "${dateSuffix}"
       addConfigureArg "--with-version-pre=" "beta"
     else
-      addConfigureArg "--without-version-opt" ""
+      # "LTS" builds (every 2 years) from jdk-21 will use "LTS" version opt
+      if [[ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" -ge 21 ]] && [[ $(((BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]-21) % 4)) == 0 ]]; then
```

Comment:
Probably nitpicking, but perhaps that logic could be extracted to a named function in case it changes

Reply:
Yeah, good idea